build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     repositories {
         jcenter()
         maven { url 'https://repo.spring.io/milestone' }
-        maven { url 'https://repo.spring.io/plugins-release' }
+        maven { url 'https://repo.spring.io/plugins-snapshot' }
```

Comment:
Rather than switching to a snapshot release, we could use maven central for pulling release versions of the dependencies.

Spring no longer offers release versions from their own  repo.spring.io, it relies on maven central

from https://spring.io/blog/2022/12/14/notice-of-permissions-changes-to-repo-spring-io-january-2023 

> repo.spring.io is intended for the download of snapshots and milestone artifacts that are not available from Maven Central.

see maven central for release versions:
https://mvnrepository.com/artifact/io.spring.gradle/propdeps-plugin